eZ/Bundle/EzPublishRestBundle/Features/Context/SubContext/Views.php

```diff
@@ -34,4 +36,16 @@ public function theSearchHitsAreContentObjects()
             Assertion::assertInstanceOf('eZ\Publish\API\Repository\Values\Content\Content', $searchHit->valueObject);
         }
     }
+
+    /**
+     * @Given /^I set contentQuery.Filter to a set of Criterion objects$/
```

Comment:
Refined the phrasing. Refined the phrasing of the sentence in b864758. The first one was more of a proof of concept. Not perfect, but suitable, I think.